helix-term/src/health.rs

```diff
@@ -189,18 +189,40 @@ pub fn languages_all() -> std::io::Result<()> {
         None => column("None", Color::Yellow),
     };
 
+    let check_binaries = |cmds: Vec<String>| {
+        match cmds.len() {
+            0 => column("None", Color::Yellow),
+            1 => check_binary(cmds.get(0).cloned()),
+            n_configured => {
+                let n_available = cmds.iter().filter_map(|cmd| which::which(cmd).ok()).count();
+                match n_available {
+                    0 => column(&format!("✘ 0/{}", n_configured), Color::Red),
+                    n_available if n_available == n_configured => {
+                        column(&format!("✓ {}/{}", n_available, n_configured), Color::Green)
+                    }
+                    n_available => column(
+                        &format!("- {}/{}", n_available, n_configured),
+                        Color::Yellow,
+                    ),
+                }
+            }
```

Comment:
```suggestion
            configured => {
                let available = cmds.iter().filter_map(|cmd| which::which(cmd).ok()).count();
                let (icon, color) =  match available {
                    0 => ("✘", Color::Red),
                    available if available == configured =>  ("✓", Color::Green),
                    available => ("-", Color::Yellow),
                }
                column(&format!("{} {}/{}", icon, available, configured), color)
            }

```

Wondering if something like this perhaps reads clearer?

Reply:
Applied in the newest commit!